pybossa/view/projects.py

```diff
@@ -1698,6 +1700,104 @@ def bulk_priority_update(short_name):
     except Exception as e:
         return ErrorStatus().format_exception(e, 'priorityupdate', 'POST')
 
+@blueprint.route('/<short_name>/tasks/assign-workersupdate', methods=['GET', 'POST'])
+def bulk_update_assign_worker(short_name):
```

Comment:
I believe this would be only admin/subadmin operation. Hence, this need to be protected with related @admin_or_subadmin_required, cors, etc...

Reply:
good point!